mart/attack/adversary.py

```diff
@@ -34,6 +34,7 @@ class Adversary(pl.LightningModule):
     def __init__(
         self,
         *,
+        disable_loading_from_state_dict: bool = True,
```

Comment:
What purpose does this serve?

Reply:
I think you would want to resume the perturbation, when resuming from training a universal perturbation.

Reply:
I don't use Adversary with universal perturbations right now.